### DISCUSSION TITLE
Template Mod with Trellis

### DISCUSSION THREAD

**Discussion by heberleh** [TIBCOSoftware/spotfire-mods#138] [Q&A] [answered]
Is there a “trellis header” builder that allows for the representation of hierarchy (lines and columns)?
Our users would also like to have only one GUI style of trellis — e.g., they click the engine, and they find the trellis menu, Title, data table, etc. in the options. Is it possible?
Any good template to start with and fit a visualization in it? I find the Violin example good, but maybe too complex to make it as an example?

**Comment by objerke** [comment, marked as answer]
Hi @heberleh, Currently there is no API for building trellis headers in visualization mods. There is currently an [idea](https://ideas.spotfire.com/ideas/TS-I-8220) in the Ideas portal that you can vote for and add your thoughts on API features.

The open source [violin plot](https://github.com/spotfiresoftware/spotfire-mod-violin) is probably the closest example at the moment.

**Comment by heberleh** [comment]
What is the current status? Any other project using trellis? Violin plot is not available anymore
